minecraftServers/Controllers/ServersController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MinecraftServers.Repositories;
+using MinecraftServers.Models;
+using System.Text.RegularExpressions;
+using MinecraftServers.Dto;
+
+namespace MinecraftServers.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ServersController : ControllerBase
+{
+    private readonly IServersRepository _serversRep;
+    private readonly Regex ip = new(@"(([01]?\d\d?|2[0-4]\d|25[0-5])\.){3}([01]?\d\d?|2[0-4]\d|25[0-5])");
+    public ServersController(IServersRepository serverRep)
+    {
+        _serversRep = serverRep;
+    }
+
+    [HttpGet]
+    public ActionResult<List<Server>> GetAll()
+    {
+        return Ok(_serversRep.GetAll());
+    }
+
+    [HttpGet("{id}")]
+    public ActionResult<Server> GetById(uint id)
+    {
+        if (_serversRep.GetById(id) == null)
+            return NotFound("We can't find this server");
+        return Ok(_serversRep.GetById(id));
+    }
+
+    [HttpPost]
+    public ActionResult<Server> Add(ServerDto server)
+    {
+        var online = 0;
+        uint id = 0;
```

Comment:
I think, there is no need to use variables. You can just pass these values as parameters